build/imports-generator.js

```diff
@@ -0,0 +1,89 @@
+/* global Promise */
+
+const fs = require('fs');
+const path = require('path');
+const browserify = require('browserify');
+const { dirname: getDirName } = require('path');
+const makeDir = require('make-dir');
+
+const importsFile = path.join(
+	__dirname,
+	'..',
+	'lib',
+	'core',
+	'imports',
+	'index.js'
+);
+
+// Mapping of external dependencies to be imported
+// Global Namespace : Library name
+const moduleMap = {
+	axios: 'axios',
+	doT: 'dot',
+	CssSelectorParser: 'css-selector-parser'
+};
+
+async function run() {
+	// generate `axe.imports` content
+	const fileContent = `
```

Comment:
I'm not in favour of dynamically generating this file. Let's just put this file in source control and have this import-generator copy over into tmp.